tutorials/clinical/10_ieeg_localize.py

```diff
@@ -0,0 +1,487 @@
+# -*- coding: utf-8 -*-
+"""
+.. _tut-ieeg-localize:
+
+========================================
+Locating Intracranial Electrode Contacts
+========================================
+
+Intracranial electrophysiology recording contacts are generally localized
+based on a post-implantation computed tomography (CT) image and a
+pre-implantation magnetic resonance (MR) image. The CT image has greater
+intensity than the background at each of the electrode contacts and
+for the skull. Using the skull, the CT can be aligned to MR-space.
+Contact locations in MR-space are the goal because this is the image from which
+brain structures can be determined using the
+:ref:`tut-freesurfer-reconstruction`. Contact locations in MR-space can also
+be translated to a template space such as ``fsaverage`` for group comparisons.
+"""
+
+# Authors: Alex Rockhill <aprockhill@mailbox.org>
+#          Eric Larson <larson.eric.d@gmail.com>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+import nibabel as nib
+from nibabel.processing import resample_from_to
+from dipy.align.imaffine import AffineRegistration, MutualInformationMetric
+from dipy.align.transforms import RigidTransform3D
+from dipy.align import (affine_registration, center_of_mass, translation,
+                        rigid, affine)
+from dipy.align.reslice import reslice
+from dipy.align.metrics import CCMetric
+from dipy.align.imwarp import SymmetricDiffeomorphicRegistration
+
+import mne
+from mne.datasets import fetch_fsaverage
+
+print(__doc__)
+
+np.set_printoptions(suppress=True)  # suppress scientific notation
+
+# paths to mne datasets - sample sEEG and FreeSurfer's fsaverage subject
+# which is in MNI space
+misc_path = mne.datasets.misc.data_path()
+sample_path = mne.datasets.sample.data_path()
+subject = 'fsaverage'
```

Comment:
We usually reserve "subject" in tutorials like this to be the individual subject, so I wouldn't use `fsaverage` here. It's only used in a couple of places anyway.
```suggestion
```

Reply:
Ok this was just copying the seeg tutorial

Reply:
In that example, there is only ever one freesurfer subject: fsaverage. Here there are two: the actual subject, and the template (fsaverage).